llm-example/llm.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import requests
+
+SYSTEM_PROMPT = os.getenv("SYSTEM_PROMPT") or ""
```

Comment:
define it as a constant somewhere. dont take this from env. You can define it as contant here itself